src/main/java/net/dv8tion/jda/api/JDABuilder.java

```diff
@@ -1410,7 +1409,7 @@ public JDABuilder setVoiceDispatchInterceptor(@Nullable VoiceDispatchInterceptor
     /**
      * The {@link ChunkingFilter} to filter which guilds should use member chunking.
      *
-     * <p>If a guild is configured for chunking the {@link #setMemberCachePolicy(MemberCachePolicy)} will be ignored.
+     * <p>Use {@link #setMemberCachePolicy(MemberCachePolicy)} to configure which members to keep in cache from chunking.
```

Comment:
```suggestion
     * <p>Use {@link #setMemberCachePolicy(MemberCachePolicy)} to configure which members to keep in cache from chunking. By default, {@link ChunkingFilter.ALL all members will be retained} when a Guild is chunked.
```

Reply:
That is not true, the defaults depend on the JDABuilder factory you used. For instance, `JDABuilder.createDefault` sets it to `NONE` and `JDABuilder.create` only sets it to `ALL`, if the `GUILD_MEMBERS` intent is enabled.

Reply:
Ah, appologies. I read the code in this method incorrectly. I incorrectly read "ChunkingFilter.All" to mean "MemberCachePolicy.All" basically.